kiosk/src/Components/GameSlide.tsx

```diff
@@ -0,0 +1,28 @@
+import { GameData } from "../Models/GameData";
+import { Kiosk } from "../Models/Kiosk";
+import HighScoresList from "./HighScoresList";
+interface IProps {
+    kiosk: Kiosk;
+    buttonSelected: boolean;
+    game: GameData;
+}
+const GameSlide: React.FC<IProps> = ({ kiosk, buttonSelected, game }) => {
+    const carouselSelected = buttonSelected ? "unselected" : "selected";
+
+    return (
+        <div className={`gameTile ${carouselSelected}`}>
+            
+            <div className="gameThumbnail" 
+                style={{backgroundImage: `url("https://makecode.com/api/${game.id}/thumb")`}}
+            />
+
+            <div className="gameDetails">
```

Comment:
nit: I'd have a preference towards just having this as 

```suggestion
            {carouselSelected && <div className="gameDetails">
```

and not having the `.gameDetails { display: none }` in the css -- a little easier to debug when the resulting DOM more closely matches what is displayed (as you don't get a bunch of elements that aren't actually there), and we already have the info here on whether it should be displayed or not anyways~

Reply:
Should it be `buttonSelected` instead of `carouselSelected`?

Reply:
whoops yeah, it didn't save this comment when I first submitted the comments & so I had to retype it and got it backwards hah

Reply:
Unfortunately, this solution won't work. We can't use `buttonSelected` because it communicates whether the `add game` button is selected or that the whole carousel is selected, not just the focused slide. If I use `buttonSelected` in this way, then the game details would not show if I highlighted the button, and I do not think this is a pattern we would want for kiosk.  The `.gameDetails { display: none }` solution is currently the only way I know to do this sort of conditional rendering since the only way to know whether or not a slide is selected is through the CSS class name `swiper-slide-active`.

I would also love to have a more React-friendly solution to this, but with how kiosk is currently designed (I didn't get a chance to do a whole system refactor), making this happen is unfortunately out of scope. 

There are a couple of solutions that I tried. 

One: I tried to do a comparison of indices. The games are stored as a list of `GameData` and the carousel has to iterate through all of the games to turn them into a `SwiperSlide`. I tried to do a comparison of `if focused slide index === current index that is iterated on, populate the slide with the game details`. This is fine for the first render but doesn't show the game details for the future selected slides because the Swiper only renders once, when the main page is loaded. I could try to trigger a rerender for the swiper each time a slide changes, but I think the performance would really tank because of this.

Two: Make The GameList listen to a hook that keeps track of when `kiosk.selectedGameIndex` changes. This is the React solution that would be pretty nice if the kiosk was set up differently. We can't use this solution because the Kiosk is a JavaScript class that does not use React at all. Because of this, we can't use state and use effect to listen to changes to `selectedGameIndex`. I have otherwise not thought of ways to listen to variable changes.

Reply:
Sounds good! as I noted it was a real nitpick so nothing blocking, nice investigation on it!